bears/general/InvalidLinkBear.py

```diff
@@ -57,24 +57,64 @@ def find_links_in_file(file, timeout, link_ignore_regex):
             [^.:%\s_/?#[\]@\\]+         # Initial part of domain
             \.                          # A required dot `.`
             (
-                (?:[^\s()%\'"`<>|\\]+)  # Path name
+                (?:[^\s()%\'\"`<>|\\]+) # Path name
                                         # This part does not allow
                                         # any parenthesis: balanced or
                                         # unbalanced.
             |                           # OR
-                \([^\s()%\'"`<>|\\]*\)  # Path name contained within ()
+                \([^\s()%\'\"`<>|\\]*\) # Path name contained within ()
```

Comment:
It's just that when I tested the regex on ``regex101.com``, It shows an error unless I escape it, Should I still remove the commit ?